src/libs/actions/BankAccounts.js

```diff
@@ -79,6 +79,9 @@ function getVBBADataForOnyx(currentStep = undefined) {
                 value: {
                     isLoading: false,
                     errors: null,
+                    // The value of some fields of the currentStep are changed i.e. being trimmed or phone number is parsed when backend returns the data
+                    // So we need to add this field in successData to update the current data from reimbursement account to the draft value of the form
```

Comment:
I'm sorry, I read this 5 times and I don't understand what it's trying to convey, can you perhaps rephrase it in other words, and with more punctuation?

Reply:
Better, maybe this is even better: 
```
// When setting up a bank account, we save the draft form values in Onyx. 
// When we update the information for a step, the value of some fields that are returned from the API 
// can be different from the value that we stored as the draft in Onyx (i.e. the phone number is formatted).
// This is why we store the current step used to call the API in order to update the corresponding draft data in Onyx.
// If currentStep is undefined that means this step don't need to update the data of the draft in Onyx.
```

Reply:
Hmmm, maybe yes?